package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "privacy-pass",
-    "version": "3.0.5",
+    "version": "3.0.6",
```

Comment:
Do you also need to bump the version here: https://github.com/privacypass/challenge-bypass-extension/blob/master/public/manifest.json#L5?

Reply:
updated